src/base/system_parameter.c

```diff
@@ -10173,6 +10193,23 @@ sysprm_generate_new_value (SYSPRM_PARAM * prm, const char *value, bool check, SY
 	    /* save size in the first position */
 	    val[0] = list_size;
 	  }
+
+	if (sysprm_get_id (prm) == PRM_ID_STORED_PROCEDURE_RETURN_NUMERIC_SIZE)
+	  {
+	    /*  
+	     *  The length of the parameter must be 2
+	     *  Check the valid range
+	     *    precision ( 1 ~ 38 ) and scale (0 ~ 38)
+	     *    precision >= scale
+	     */
+	    if (val[0] != 2 || val[1] > DB_MAX_NUMERIC_PRECISION || val[1] < 1 || val[2] > DB_MAX_NUMERIC_PRECISION
+		|| val[2] < 0 || val[1] < val[2])
```

Comment:
define 된 문자열을 사용하는 것을 추천 드립니다.

```suggestion
	    if (val[0] != 2 || val[PRM_PRECISION] < val[PRM_SCALE]
	        || val[PRM_PRECISION] > DB_MAX_NUMERIC_PRECISION  ||  val[PRM_PRECISION] < 1 
	        || val[PRM_SCALE] > DB_MAX_NUMERIC_PRECISION  || val[PRM_SCALE] < 0 )
```